tests/platform_tests/counterpoll/counterpoll_constants.py

```diff
@@ -0,0 +1,41 @@
+
+class CounterpollConstants:
+    COUNTERPOLL_SHOW = 'counterpoll show'
+    COUNTERPOLL_DISABLE = 'counterpoll {} disable'
+    COUNTERPOLL_RESTORE = 'counterpoll {} {}'
+    COUNTERPOLL_INTERVAL_STR = 'counterpoll {} interval {}'
+    COUNTERPOLL_QUEST = 'counterpoll ?'
```

Comment:
Suggest use "counterpoll --help" which is more stable and easy to read.

Reply:
fix